openapi-filter.js

```diff
@@ -47,10 +47,33 @@ let argv = require('yargs')
     .number('maxAliasCount')
     .default('maxAliasCount',100)
     .describe('maxAliasCount','maximum YAML aliases allowed')
+    .alias('configFile', 'c')
+    .describe('configFile', 'The file & path for the filter options')
+    .boolean('verbose')
```

Comment:
Ditto.

Reply:
I have changed it the verbose parameter

```
    .alias('verbose', 'v')
    .count('verbose')
    .describe('verbose', 'Output more details of the filter process. Change verbose level with -v -vv or -vvv')
```

But it seems that the args does properperly count 

`console.log('argv.verbose', (argv.verbose))` results in `argv.verbose []`

When using 
```
 node openapi-filter.js ./test/configFile/input.yaml ./test/configFile/output.yaml -vvv --configFile ./test/configFile/options.json
```

Reply:
This seems to work fine with the same version of `yargs` as we have in the project:

```shell
mike:~/node/t2$ cat t.js
const argv = require('yargs')
  .count('verbose')
  .describe('verbose','increase verbosity')
  .alias('v','verbose')
  .argv;

console.log('argv.verbose',argv.verbose);
mike:~/node/t2$ node t.js -vvv
argv.verbose 3
```

Reply:
@MikeRalphson 

I found the cause:  `.alias('flagValues', 'v')` which overlaps with `.alias('v','verbose')`

From

```
    .array('flagValues')
    .alias('flagValues', 'v')
    .describe('flagValues', 'flag string values to consider as a filter match (in addition to matching on boolean types)')
    .default('flagValues', [])
```

I could remove the "verbose" part of out the PR or if you have a proach to use -v for which operation?
Not sure what do you prefer to solve this?

Reply:
Let's drop the -v alias for --verbose, as we don't want to break any existing usage of -v in a minor release.

Reply:
Solved.

`node openapi-filter.js ./test/configFile/input.yaml ./test/configFile/output.yaml --configFile ./test/configFile/options.json --verbose`

Results in: 
```
=== Document filtering started ===

CONFIG File:  ./test/configFile/options.json
IN File:      ./test/configFile/input.yaml
OUT File:     ./test/configFile/output.yaml

✅ Document was filtered successfully!

```

Without the --verbose option, nothing is output.

I added a little helper function to make the code cleaner and easier to maintain.